tests/tests.js

```diff
@@ -92,7 +96,9 @@ var runTests = function runTests(arrayFrom) {
 	});
 
 	test('throws with an invalid mapping function', function (t) {
-		t.throws(function () { arrayFrom([], undefined); }, TypeError);
+		t.doesNotThrow(function () { arrayFrom([], undefined); }, TypeError);
```

Comment:
for `doesNotThrow`, we want to assert it throws nothing, not just that it doesn't throw a TypeError - can we remove the exception type?